src/main/java/org/sagebionetworks/bridge/sdk/UserSession.java

```diff
@@ -26,17 +23,15 @@
     private final SharingScope sharingScope;
     private final boolean authenticated;
     private final String id;
-    @JsonDeserialize(using=DataGroupsDeserializer.class)
-    @JsonSerialize(using=DataGroupsSerializer.class)
-    private final DataGroups dataGroups;
+    private final Set<String> dataGroups;
```

Comment:
Rewrote the UserSession test which was not up-to-date to verify all the stuff in user session is deserialized correctly from session JSON.